db/seeds.rb

```diff
@@ -1,16 +1,48 @@
 # Создадим записи в базе данных для проверки,
 # восклицательный знак после create стоит для вызова исключения.
+# encoding: utf-8
+require 'open-uri'
+require 'nokogiri'
 
-Card.create!([
-  { original_text: "Привет. Как дела?",
-    translated_text: "Hello. How are you?",
-    review_date: Date.current },
+class WordsDownloader
+  def initialize(url, original_text_col=2, translated_text_col=3)
+    @url = url
+    @original_text_col = original_text_col
+    @translated_text_col = translated_text_col
+  end
 
-  { original_text: "Откуда ты?",
-    translated_text: "where are you from?",
-    review_date: Date.yesterday },
+  def get_words(&closure)
+    get_rows() if @rows == nil
+    words = []
+    closure = lambda { |row| return row.css('td') } || lambda { closure }
+    @rows.each do |row|
+      cols = closure.call(row)
+      words << [cols[@original_text_col].text, cols[@translated_text_col].text]
+    end
+    return words
+  end
+
+
+
+  def get_rows(&closure)
+    doc = Nokogiri::HTML(open(@url))
+    closure = lambda { return doc.css('table.topwords').css('tr')[1..-1] } || lambda { closure }
```

Comment:
Line is too long. [96/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.